Automattic/newspack-theme
# Remove non-latin font fallbacks

### All Submissions:


### Changes proposed in this Pull Request:

This PR removes the non-latin font fallbacks from the theme. Though they make sense in a default theme, like Twenty Nineteen, Newspack's own customization options should make these fallbacks unnecessary. It also shaves 2000+ lines of CSS from the theme.

Closes #9.

### How to test the changes in this Pull Request:

1. Apply the PR. 
2. Run the build script (`npm run build`) to make sure the CSS compiles without errors.
3. View the front-end and editor, to confirm the typography is still styled correctly.

### Other information:
